docs/architecture/ADR-001-uToken-interest.md

```diff
@@ -0,0 +1,92 @@
+
+Behavior over IBC (and Ethereum) bridges can be expected to follow this pattern, where the _amount_ of uTokens transferred is represented faithfully, but any complex _behavior_ of uTokens is lost.
+
+## Decision
+
+"uToken to base asset exchange rate grows over time". This method is inspired by the Compound model, as illustrated in [the example found here](https://compound.finance/docs/ctokens#introduction)
+
+- Requirement: Umee chain maintains `Token:Utoken` exchange rate for each token
+
+In this implementation, for each token type accepted by the `x/leverage` module, the Umee chain computes an "exchange rate" between the base asset and its associated uToken. The exchange rate starts equal to 1, and increases as interest accrues. Whenever a lender deposits or withdraws base assets for uToken, the current exchange rate is used.
+
+Example scenario:
+
+> Two lenders Alice and Bob provide `ATOM` to the asset facility at different times and earn interest. Assume that for the duration of this scenario, the interest on deposited uAtoms is 0.1 percent per week (or `1 ATOM` per week per `1000 ATOM` deposited).
+>
+> The asset facility starts with `0 ATOM` in custody and `0 u/ATOM` in circulation. The `u/ATOM` exchange rate starts at `1.0`.
+>
+> At a time we will label week=0, Alice deposits `2000 ATOM` and receives `2000 u/ATOM` per the exchange rate.
+>
+> By week=1, the `u/ATOM` exchange rate has increased to `1.001` due to interest. Alice's `2000 u/ATOM` are now worth `2002 ATOM` if redeemed.
+>
+> By week=2, the `u/ATOM` exchange rate is `1.002` (technically `1.002001` due to compound interest, but approximate amount will be displayed here)
+>
+> Also at week=2, Bob deposits `1000 ATOM`. Because the exchange rate has shifted, he receives `998 u/ATOM`,  worth `1000 ATOM` if redeemed immediately.
+>
+> By week=3, the `u/ATOM` exchange rate is `1.003`. Bob's `998 u/ATOM` are now worth `1001 ATOM` if redeemed. Alice's `2000 u/ATOM` are worth `2006 ATOM`.
+>
+> By week=4, the `u/ATOM` exchange rate is `1.004`.
+>
+> Also at week=4, Alice redeems `1000 u/ATOM` for `1004 ATOM` per the exchange rate.
+>
+> Also at week=4, Bob deposits an additional `1000 ATOM`, and receives `996 u/ATOM` using the current exchange rate. His `u/ATOM` balance of 998+996 = `1994 u/ATOM` is worth 1002+1000 = `2002 ATOM`, as the two parts have grown by 0.2% and 0% respectively.
+
+This implementation sacrifices the "1:1 uToken to base asset exchange rate" and "uToken balances grow over time" facts promised in the whitepaper, while maintaining a mathematically identical incentive structure. In exchange, IBC and ERC20 transfer of uTokens becomes possible without stopping interest. A uToken's value increases no matter where it is held, by virtue of the Token:uToken exchange rate.
```

Comment:
could you add a motivation why this model was selected over accumulating the interest rate in the balance?

Reply:
I'll split this paragraph and clarify the last sentence, to indicate that tracking and minting uTokens on other chains was prohibitive. That, or I can add back the `Alternatives` subsection since. it's warranted.

Reply:
yes, but it's not the syntax of the denom we construct.

Reply:
I considered saying something like "u/tokens" in documentation (to mimic our actual denom, which acts like `ibc/` and `gravity/` denoms) but decided against proposing it. As far as docs go, "uTokens" is unlikely to mislead.

Reply:
OK, won't block here, but my preference is to stick with what we have in the code - the denom we will show to the users. Maybe we can ask others if they have a preference before merging this? I think it's an important decision, and we will need to stick with it in all other documentation pieces.